azure-samples/src/main/java/com/microsoft/azure/management/compute/samples/ListVirtualMachineImages.java

```diff
@@ -56,8 +56,8 @@ public static boolean runSample(Azure azure) {
             System.out.println("Publisher - " + publisher.name());
 
             if (publisher.name().equalsIgnoreCase("Canonical")
-                    | publisher.name().equalsIgnoreCase("Suse")
```

Comment:
Looks like incorrect logical OR condition.
I guess it should be 'II' and not 'I'

Reply:
[P2]: Nice catch, I marked it as P2 as it is a fix needed in sample. Thanks for finding this.